app/src/main/res/values/strings.xml

```diff
@@ -1206,6 +1206,12 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="achievement_lifesaver_solved_X">You solved %d quests that help emergency services and first responders!</string>
     <string name="quest_airConditioning_no_name_title">"Does this place (%s) have air conditioning?"</string>
     <string name="quest_airConditioning_name_type_title">"Does %1$s (%2$s) have air conditioning?"</string>
+    <string name="quest_smoking_no_name_title">"Is smoking allowed anywhere in this place (%s)?"</string>
+    <string name="quest_smoking_name_type_title">"Is smoking allowed anywhere in %1$s (%2$s)?"</string>
+    <string name="quest_smoking_no">No, smoking is forbidden</string>
+    <string name="quest_smoking_outside">Only allowed outside</string>
+    <string name="quest_smoking_separated">Allowed in some areas, but not in others</string>
+    <string name="quest_smoking_yes">Yes, smoking is allowed everywhere</string>
```

Comment:
```suggestion
    <string name="quest_smoking_no">No, smoking is forbidden</string>
    <string name="quest_smoking_outside">Only allowed outside</string>
    <string name="quest_smoking_separated">Allowed in separated area</string>
    <string name="quest_smoking_yes">Yes, smoking is allowed everywhere</string>
```
I find this clearer to be honest.

Also, I haven't followed the discussion so far, but is there any reason why the answers need to be that wordy? the more words, the less people read it. The current answers kind of repeat the question.
```suggestion
    <string name="quest_smoking_no">No, forbidden</string>
    <string name="quest_smoking_outside">Only outside</string>
    <string name="quest_smoking_separated">Only in separated area</string>
    <string name="quest_smoking_yes">Yes, everywhere</string>
```